apps/molgenis-components/src/components/forms/formUtils/formUtils.ts

```diff
@@ -28,6 +28,10 @@ function getColumnError(
   // FIXME: longs are not checked
   const missesValue = value === undefined || value === null || value === "";
 
+  if (!isColumnVisible(column, values, tableMetaData)) {
```

Comment:
should this code be at line 24? Because now we still run isInvalidNumber?

Reply:
Doesn't really matter, since isInvalidNumber does not cause a return yet. This way is a bit more expensive, but I think it is more readable this way  (first declaring al vars, then doing al the actual return logic)